soroban-sdk/src/env.rs

```diff
@@ -300,6 +300,17 @@ impl Env {
         Crypto::new(self)
     }
 
+    /// Get a [Prng] for accessing the current functions which provide pseudo-randomness.
```

Comment:
I think we should mention some additional important properties somewhere, such as:

- every contract invocation gets its own, independent seed, thus it is not possible to manipulate prng of another contract
- prng is not replayable, if an invocation fails, next invocation will still get a different seed.
- The rng results are observable immediately after the transaction is included into ledger (that's mentioned in the file, but I think it's important enough to also be mentioned at the top level)

Feel free to edit/modify this as you see fit. Also I realize that SDK is probably not the best place to document the network-dependent details, but OTOH I'm not sure what else is.

Reply:
Thanks, great call out. I'll add these details into the module doc at the top.

> I realize that SDK is probably not the best place to document the network-dependent details

I was wondering this too. I added a note at the top that is really Stellar specific. I'll play with that and see if I can remove it.

Reply:
Pushed updated docs. Lmk if you think anything should be changed.